### ISSUE TITLE
Increase max playlist size

### ISSUE THREAD

**Issue by miek770** [IrosTheBeggar/mStream#236]
Hi,

This is related to #179 but without the download aspect, hence the new issue. I would like to be able to create playlists of more than whatever the current song limit is set to (appears to be 2k based on #179, or was that only for downloads?).

When I try to save a big playlist, nothing happens after I click on *Save playlist* in the popup. The popup stays open, and there's no error in the server log. I got a *PayloadTooLargeError* but I think that was when I tried sending the whole library to the playlist (didn't try reproducing it because I don't want to lose my currently unsaveable playlist).

@IrosTheBeggar mentioned:
> That's not surprising. The server is actually rejecting the message due to the size. But it would probably crash due to a memory error if that message made it through.

I assume this relates to the download, not the playlist size.

Thanks.

**Comment by xrayspx** [comment]
I just did some testing, and I don't think there's an arbitrary numerical limit on the number of songs.  I took a folder with 143 songs in it, and was able to create a playlist of 1144 songs and save it, however if I added the folder again and tried to save with 1287 tracks, it failed with "Call Failed".  Starting with my 1144 playlist, I added songs 10 at a time from another folder, and was able to pass that 1287 limit and create a playlist of 1294.  So it looks like it may be something else limiting it possibly?
